app/helpers/moment-add.js

```diff
@@ -0,0 +1,10 @@
+import helper from 'ember-helper';
+import moment from 'moment';
+
+export function momentAdd(params) {
+  const [dateAsMoment, addValue, addUnit, outputFormat] = params;
+  const newMoment = moment(dateAsMoment).clone().add(addValue, addUnit).format(outputFormat);
+  return newMoment;
+}
+
+export default helper.helper(momentAdd);
```

Comment:
Question about template helpers vs. controller - the full day & date (e.g. Sunday, January 1, 2017) is used in both the template and also the `titleToken` attr on the route. Is there any place other than the controller I can write that format logic once and use in both places, or should I plan on just formatting that string independently in separate files?